web/frontend/src/components/slam.vue

```diff
@@ -108,6 +196,13 @@ const refresh2d = async (name: string) => {
 const handleRefresh2dResponse = (response: MapAndPose): void => {
   pointcloud = response.map;
   pose = response.pose;
+  basePose.setX(Number(pose!.getX()!.toFixed(1)!))
```

Comment:
Why is this needed?

Reply:
base pose is used to display the current orientation and position of the rover.
we do `.toFixed(1)! ` to round to the tenths.

Reply:
Why do we need to round to the tenths? Is this purely for aesthetic reasons? If so, could we please add that in a comment?

Reply:
we round to the tenths since that is what is on the figma design card.

Reply:
cool, could we get a comment?